projects/app/js/restore.test.js

```
@@ -546,4 +547,45 @@ describe('RestoreManager - custom animations and configured directory handles',
+        const settingsContent = buildSettingsJson([]);
+        // Ndjson with a history task log missing syncId and updatedAt
+        const logContent = JSON.stringify({
+            kind: 'QuickLogSolo/History',
+            version: '1.0',
+            type: 'task',
+            startTime: 1700000000000,
+            endTime: 1700000060000,
+            category: 'Dev',
+            color: 'primary',
+            tags: ['tag1'],
+            memo: 'some memo'
```

ESLint

[error] Insert `,`

(prettier/prettier)